.github/workflows/workflow_pr.yml

```diff
@@ -476,3 +476,16 @@ jobs:
           echo "${{ needs.workspace_name.outputs.name }} PR environment tested, built and deployed"
           echo "Tag Deployed: ${{ needs.image_tag.outputs.short_sha }}"
           echo "URL: https://${{ env.FRONT_URL }}/home"
+
+  test_delete_env:
```

Comment:
What's the purpose of this task?

Reply:
It's going to be removed in another commit. I just want to make sure it works before it gets merged into main